commons/src/main/java/gov/cms/qpp/conversion/DocumentationReference.java

```diff
@@ -6,7 +6,8 @@
 	CLINICAL_DOCUMENT(19),
 	PRACTICE_SITE_ADDRESS(25),
 	REPORTING_PARAMETERS_ACT(80),
-	MEASURE_IDS(88);
+	MEASURE_IDS(88),
+	PERFORMANCE_PERIOD(17);
 
 	private static final String BASE_PATH = "https://ecqi.healthit.gov/system/files/eCQM_QRDA_EC-508_0.pdf#page=";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)